packages/edit-site/src/components/table/style.scss

```diff
@@ -0,0 +1,39 @@
+.edit-site-table-wrapper {
+	overflow: auto;
```

Comment:
Probably better to move this into `.edit-site-page-content` to avoid the hidden focus ring:

![image](https://github.com/WordPress/gutenberg/assets/7753001/af8f4f0a-d626-4ae1-ac2f-d58acd86ee25)

Reply:
Nice pickup, adjusted.